cmd/datasource-syncer/README.md

```diff
@@ -29,6 +29,7 @@ DATASOURCE_UIDS=YOUR_DATASOURCE_UIDs
 GRAFANA_API_TOKEN=YOUR_GRAFANA_SERVICE_ACCOUNT_TOKEN
 GRAFANA_API_ENDPOINT=YOUR_GRAFANA_INSTANCE_URL
 PROJECT_ID=PROJECT_ID_TO_QUERY_GCM
+NAMESPACE_NAME=NAMESPACE_NAME that is binded to your Google Cloud service account.
```

Comment:
would get rid of the help text as it probably will make an error if copy-pasted

Could put it below, "choose the namespace in which you deploy the data source syncer. This should be the same namespace you bind your workload identity service account to, if enabled"

Reply:
```suggestion
NAMESPACE_NAME=NAMESPACE_NAME #that you deploy the datasource-syncer to, and that you bound your service account to if using workload identity
```